### PR TITLE
Bug 1133826 - Private window styles in task manager

### PR DESCRIPTION
**Pull request by sfoster**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[sfoster Sam Foster (sfoster)](https://github.com/sfoster) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f7d895ef345f)

**Comment by try-server-hook** [comment]
[sfoster Sam Foster (sfoster)](https://github.com/sfoster) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7e8d41c352b4)

File: apps/system/style/cards_view/cards_view.css [hunk 1]
```diff
@@ -122,6 +122,21 @@
   transform-origin: 50% 50%;
 }
 
+#cards-view .card.private .screenshotView:after {
```
Comment (etiennesegonzac) [inline apps/system/style/cards_view/cards_view.css]: nit: `::after`

File: apps/system/style/cards_view/cards_view.css [hunk 2]
```diff
@@ -122,6 +122,21 @@
...
+  content: "";
+  position: absolute;
+  left: 0; top: 0;
+  background-color: rgba(57, 46, 84, 0.9);
+  background-image: url(../icons/pb_icon.png);
+  background-repeat: no-repeat;
+  background-size: 50%;
+  background-position: center center;
+  opacity: 1;
+  transition: opacity 0.4s ease;
```
Comment (etiennesegonzac) [inline apps/system/style/cards_view/cards_view.css]: I'm pretty sure we'll never see this transition. We should remove it.

(we can maybe have a transition by making the opacity:0 here and 1 when `#cards-view.active` but I'm not sure if it'll perform)

**Comment by try-server-hook** [comment]
[sfoster Sam Foster (sfoster)](https://github.com/sfoster) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5450e5765149)

**Pull request merged**